sources/platform/monitoring/index.md

```diff
@@ -1,94 +1,108 @@
+    ![Daily run statuses](./images/daily-run-statuses.png)
 
-## How does it work? {#how-does-it-work}
+2. Chart displaying **metrics** of the last 200 runs of the actor or saved task.
+    ![Run statistics](./images/run-statistics-chart.png)
 
-The monitoring suite is made up of a system of actors: **checkers** and **reporters**. Based on your configuration, the suite will create [schedules](../schedules.md) and [webhooks](../integrations/webhooks/index.md) under your account and use those to run these actors.
+3. Option to set up **alerts** with notifications based on the run metrics.
+    ![Alerts](./images/alerts.png)
 
-As the names suggest, the checkers make sure your data is up to standard. Reporters create visualizations of how your actors perform over time and send [notifications](#notifications) about failed runs.
+> Both charts can also be added to your Apify Console home page so that you can quickly see if there are any issues every time you open the Apify Console.
 
-![Monitoring dashboard](./images/monitoring-dashboard.png)
+### Alert configuration
 
-## How much does it cost? {#how-much-does-it-cost}
+When you set up an alert you have two choices of how you want the metrics to be evaluated. And depending on your choices the alerting system will behave differently:
 
-You can use the monitoring suite **free of charge**, but under the hood, it is a system of actors that consume [compute units](https://help.apify.com/en/articles/3490384-what-is-a-compute-unit) (CUs). We have tried our best to make it as efficient as possible.
+1. **Alert, when the metric is lower than** - This type of alert is checked after the run finishes. If the metric is lower than the value you set, the alert will be triggered and you will receive a notification.
 
-* Medium projects (single monitoring configuration with 5-20 daily running monitored tasks or actors) should consume around **10 CUs per month**.
-* Bigger projects (more monitoring configurations with more than 20 daily running tasks or actors) can exceed **50 CUs+ per month**.
+2. **Alert, when the metric is higher than** - This type of alert is checked both during the run, and after the run finishes. During the run we are doing periodic checks (approximately every 5 minutes) so that we can notify you as soon as possible if the metric is higher than the value you set. After the run finishes we are doing a final check to make sure that the metric did not change over the limit in the last few minutes of the run.
 
-## How can I set up monitoring? {#how-can-i-set-up-monitoring}
+![Metric condition configuration](./images/metric-options.png)
 
-We currently have video tutorials for setting up monitoring and adding data validation.
+You can get notified by email, Slack, or in the Apify Console. If you use Slack, we suggest using Slack notifications instead of email, because they are more reliable and you can also get notified quicker.
```

Comment:
```suggestion
You can get notified by email, Slack, or in Apify Console. If you use Slack, we suggest using Slack notifications instead of email because they are more reliable, and you can also get notified quicker.
```